shared/output/gapic/templates/vision/google/cloud/vision/v1/image_annotator.rb

```diff
@@ -119,12 +119,12 @@ def initialize \
 
             defaults = default_settings client_config, timeout, metadata, lib_name, lib_version
 
-            @batch_annotate_images = Google::Gax.create_api_call(
+            @batch_annotate_images = Google::Gax::ApiCall.new(
```

Comment:
This is still here because only the cloud templates were updated - correct (and same for other test clients)?

Reply:
Correct. This file shouldn't have been changed, but its going away soon so I'm not too worried about it.